### PR TITLE
makes aggregate report form behavior consistent with other forms

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: webapp/src/main/webapp/src/app/aggregate-report/query-forms/aggregate-query-form-container.component.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <ng-template #submitButtonTemplate>
   <button
     class="btn btn-success btn-sm green"
-    [disabled]="formByReportType[queryType].submissionSubscription"
+    [disabled]="formByReportType[queryType].userReportSubscription"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/aggregate-report/query-forms/aggregate-query-form-container.component.html]: renamed for consistency with other forms

File: webapp/src/main/webapp/src/app/aggregate-report/query-forms/aggregate-query-form-container.component.html [hunk 2]
```diff
@@ -70,33 +70,32 @@ <h3>
         </div>
       </div>
 
-      <ng-container *ngFor="let typeString of queryTypes">
+      <div
+        *ngFor="let typeString of queryTypes"
+        [hidden]="queryType !== typeString"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/aggregate-report/query-forms/aggregate-query-form-container.component.html]: brought this up to eliminate the redundancy

File: webapp/src/main/webapp/src/app/aggregate-report/query-forms/base-aggregate-query-form.component.ts [hunk 3]
```diff
@@ -87,7 +85,8 @@ export abstract class BaseAggregateQueryFormComponent
   );
 
   protected subjectDefinitions: SubjectDefinition[] = [];
-  private initialQuery: AggregateReportQueryType;
+
+  initialQuery: AggregateReportQueryType;
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/aggregate-report/query-forms/base-aggregate-query-form.component.ts]: not private anymore so the view can access it - this is used to know if a user query is loaded or not

File: webapp/src/main/webapp/src/app/aggregate-report/query-forms/base-aggregate-query-form.component.ts [hunk 4]
```diff
@@ -164,33 +162,35 @@ export abstract class BaseAggregateQueryFormComponent
       this.updateSubjectsEnabled();
       this.setDefaultAssessmentType();
       this.initialize();
-      this.initialQuery = {
-        ...this.createReportRequest(),
-        type: query.type
-      };
+      if (userQueryId != null) {
+        this.initialQuery = {
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/aggregate-report/query-forms/base-aggregate-query-form.component.ts]: we only need this for diffing if a user query is loaded

File: webapp/src/main/webapp/src/app/aggregate-report/query-forms/claim-report-form.component.html [hunk 5]
```diff
@@ -735,11 +735,16 @@ <h3>
                 </p>
               </div>
             </div>
-            <div>
+            <div *ngIf="initialQuery != null">
               <br />
-              <ng-container
-                [ngTemplateOutlet]="saveQueryTemplate"
-              ></ng-container>
+              <button
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/aggregate-report/query-forms/claim-report-form.component.html]: made this change to all 4 aggregate report form templates

File: webapp/src/main/webapp/src/app/aggregate-report/query-forms/aggregate-query-form-container.component.html (hunk 2 above)
Comment (malaffoon) [reply webapp/src/main/webapp/src/app/aggregate-report/query-forms/aggregate-query-form-container.component.html]: Yeah redundancy sucks. And so does redundancy.

**Pull request merged**